### ISSUE TITLE
Wikipedia Retriever missing dictionary

### ISSUE THREAD

**Issue by JohannesTK** [facebookresearch/ParlAI#1594]
Both [Wikipedia Retriever (Used For Open Squad)
](http://parl.ai.s3-website.us-east-2.amazonaws.com/docs/zoo.html#wikipedia-retriever-used-for-open-squad) and [Wikipedia Retriever (Used For Wizard Of Wikipedia)
](http://parl.ai.s3-website.us-east-2.amazonaws.com/docs/zoo.html#wizard-of-wikipedia-end-to-end-generator) seem to be missing their dictionaries. 

```
python -m parlai.scripts.interactive --model tfidf_retriever -mf models:wikipedia_2016-12-21/tfidf_retriever/drqa_docs
[building data: /home/ubuntu/ParlAI/data/models/wikipedia_2016-12-21/tfidf_retriever/drqa_docs.tgz]
[ downloading: http://parl.ai/downloads/_models/wikipedia_2016-12-21/tfidf_retriever/drqa_docs.tgz to /home/ubuntu/ParlAI/data/models/wikipedia_2016-12-21/tfidf_retriever/drqa_docs.tgz ]
Downloading drqa_docs.tgz: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 7.72G/7.72G [05:06<00:00, 25.2MB/s]
unpacking drqa_docs.tgz
[ warning: overriding opt['model_file'] to /home/ubuntu/ParlAI/data/models/wikipedia_2016-12-21/tfidf_retriever/drqa_docs (previously: wiki_full_notitle )]
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/pytorch_p36/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/ubuntu/anaconda3/envs/pytorch_p36/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/ubuntu/ParlAI/parlai/scripts/interactive.py", line 76, in <module>
    interactive(parser.parse_args(print_args=False), print_parser=parser)
  File "/home/ubuntu/ParlAI/parlai/scripts/interactive.py", line 54, in interactive
    agent = create_agent(opt, requireModelExists=True)
  File "/home/ubuntu/ParlAI/parlai/core/agents.py", line 552, in create_agent
    model = load_agent_module(opt)
  File "/home/ubuntu/ParlAI/parlai/core/agents.py", line 403, in load_agent_module
    'is correct'.format(old_dict_file, new_opt['dict_file'])
RuntimeError: WARNING: Neither the specified dict file (test_ret.dict) nor the `model_file`.dict file (/home/ubuntu/ParlAI/data/models/wikipedia_2016-12-21/tfidf_retriever/drqa_docs.dict) exists, check to make sure either is correct
```

**Comment by stephenroller** [comment]
Blah, this is a runaway check; those models don't need dicts. You can comment out that RunTime error and continue without problem.

**Comment by emilydinan** [comment]
Fixed by #1613
